### DISCUSSION TITLE
Push Button for IKEA Jetstrom

### DISCUSSION THREAD

**Discussion by theincogtion** [Koenkk/zigbee2mqtt#21883] [Devices]
Hi guys I am currently using ZHA but will migrate to zigbee2mqtt in Home Assistant.
I own a jetstrom ikea lamp which I can control via zigbee. The lamp needs continous power supply.
I have 2 options now:
### Buying a relay
I could use a relay in combination with my push button switch and only monitor the input of the switch without connecting the lamp to the relay.
Which devices are able to work with a push button and can dimm my light with a long press?
MoesGo ZigBee from german amazon supports a push button but I couldn't find it on the compatibility list here.

### Buying a wall switch
I could also buy a switch (with neutral) and use its inputs to control the lamp. Same question here:
Can I use it to dimm it by holding down a button?
I checked the compatibility list and found this one: ZB-SW02
With this name I can't find it on german amazon nor on aliexpress.
Any ideas where to buy it or an alternative device which would work?

**Comment by chris-1243** [comment]
Devices from Ubisys like a C4

https://www.zigbee2mqtt.io/devices/C4.html

Or a S1/S2 (relay)

https://www.zigbee2mqtt.io/devices/S1.html

https://www.zigbee2mqtt.io/devices/S2.html

As it seems you are in Germany, it won't be difficult to find these products. They have one drawback, the price...

I do use two S2 and two C4 and they work perfectly
